src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -97,7 +98,7 @@ public Entity retrieveLocationEntity(String id){
 
   public void addLocationCountByOne(Entity locationEntity) {
     int count = (int)(long)locationEntity.getProperty("count");
-    locationEntity.setProperty("count", count++);
+    locationEntity.setProperty("count", ++count);
```

Comment:
I would think that the count represents the number of posts we have at this location, so this shouldn't be a count of how many locations there are?